test/test_web.py

```diff
@@ -23,7 +20,7 @@
     pass
 
 from . import tools
-from .tools import browser, get_with_retry, WAIT_TIME, locked_cache_dir, WIN
+from .tools import browser, get_with_retry, WAIT_TIME, locked_cache_dir, WIN  # noqa F401 '.tools.browser' imported but unused. The fixture 'browser' is necessary for several functions
```

Comment:
Do you mind moving the fixture of fixtures to conftest in a separate PR before merging this, so we don't need to have this `noqa`.

CC: @dorothykiz1

Reply:
I worked on this